INSTALL.md

```diff
@@ -164,6 +164,18 @@ Then you can try one of the
 [examples](https://github.com/kubernetes/client-go/tree/v2.0.0/examples/) from
 the 2.0.0 release.
 
+For client-go 4.0.0, a k8s.io/apimachinery constraint must be defined in Gopkg.toml
+
+```
+[[constraint]]
+  name = "k8s.io/client-go"
+  version = "4.0.0"
+
+[[constraint]]
```

Comment:
This should be [`[[override]]`](https://github.com/golang/dep/blob/master/docs/Gopkg.toml.md#override), as `apimachinery` is a transitive dependency (though, of course, this project should have it's own lockfile)

Reply:
Yes, it should be override. But still we are missing a ton of dependencies to be locked to our supported version (compare https://github.com/sttts/kube-vendor-test for the minimal list to get some binaries to compile). We really need this Gopkg.toml in our repos, and we will for 1.8. This is kind of blocked by https://github.com/kubernetes/kubernetes/issues/51643 right now to go forward with that.